controller/helper/process-survey-instances.js

```diff
@@ -12,17 +12,48 @@ const moment = require('moment');
  * @returns {Object} Complience chart data
  */
 function processSurveyInstances (surveys) {
+    const filterSurveyByState = surveys.filter((survey) => {
+        return survey.state === 'completed';
+    });
+
+    const surveyInstanceLength = 0;
+
+    if (filterSurveyByState !== 'undefined' && filterSurveyByState.length > surveyInstanceLength) {
```

Comment:
`surveys.filter` will always return an array, `pickDates` and `pickTimeLeft` both use `Array.map` which can gracefully handle an empty array.
So this check should be unnecessary.